extensions/resteasy-reactive/quarkus-resteasy-reactive-jsonb/deployment/src/main/java/io/quarkus/resteasy/reactive/jsonb/deployment/processor/ResteasyReactiveJsonbProcessor.java

```diff
@@ -22,11 +21,11 @@
 
 public class ResteasyReactiveJsonbProcessor {
 
-    private static final List<String> VERTX_SERIALIZERS = Arrays.asList(
+    private static final List<String> VERTX_SERIALIZERS = List.of(
             VertxJson.JsonObjectSerializer.class.getName(),
             VertxJson.JsonArraySerializer.class.getName());
 
-    private static final List<String> VERTX_DESERIALIZERS = Arrays.asList(
+    private static final List<String> VERTX_DESERIALIZERS = List.of(
```

Comment:
Isn't it making the List mutable?

Reply:
Nope, that's the magic of `List.of` - it returns an immutable list

Reply:
Awesome, I need to update my collections skills :)

Reply:
It's crazy how Java didn't have these all these years...